kuma/javascript/src/document-provider.jsx

```diff
@@ -67,7 +67,7 @@ export default function DocumentProvider(
         // This is the function that does client side navigation
         function navigate(url, localeAndSlug) {
             body.style.opacity = '0.15';
-            fetch(`/api/v1/doc${localeAndSlug}`)
+            fetch(`/api/v1/doc${localeAndSlug}`, { redirect: 'follow' })
```

Comment:
I don't think you need the `redirect: 'follow'`. It's done by default.
See https://codepen.io/peterbe/pen/pBxvYM

Reply:
From MDN (https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/fetch) it seems like it's the default in Chrome, but maybe not in other browsers? Anyway, I thought it was safest to be explicit.

Reply:
I ran my Codepen above in Firefox Nightly and it implied that `{ redirect: 'follow' }` was *not* needed. I also opened that codepen in Safari and it behaves the same. 

But perhaps best to leave it in case there was an initial time when it was *not default* and that might get stuck in some ancient browsers.